index.js

```diff
@@ -0,0 +1,209 @@
+  },
+};
+
+module.exports = {
+  extends: ['airbnb', 'plugin:prettier/recommended'],
+  plugins: ['import', 'unicorn', 'babel'],
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  env: {
+    browser: true,
+    es6: true,
+  },
+  globals: {
+    require: true,
+    process: true,
+  },
+  rules: {
+    'babel/no-unused-expressions': 'error',
+    'default-case': 'error',
+    'import/extensions': [
+      'error',
+      'always',
+      { js: 'never', jsx: 'never', ts: 'never', tsx: 'never', hbs: 'never', vue: 'never' },
+    ],
+    'import/no-unresolved': 'error',
+    'import/order': ['error', { groups: ['external', 'builtin', ['sibling', 'parent']] }],
+    'import/prefer-default-export': 'off',
+    'lines-between-class-members': 'off',
+    'max-lines': 'error',
+    'no-console': 'error',
+    'no-debugger': 'error',
+    'no-extra-boolean-cast': 'off',
+    'no-plusplus': 'off',
+    'no-restricted-properties': [
+      'error',
+      { object: 'document', property: 'querySelector' },
+      { object: 'document', property: 'querySelectorAll' },
+    ],
+    'no-undef': 'error',
+    'no-underscore-dangle': 'error',
+    'no-unused-expressions': 'off',
+    'react/static-property-placement': 'off',
+    'unicorn/catch-error-name': 'error',
+    'unicorn/custom-error-definition': 'off',
+    'unicorn/error-message': 'error',
+    'unicorn/escape-case': 'error',
+    'unicorn/explicit-length-check': 'error',
+    'unicorn/new-for-builtins': 'error',
+    'unicorn/no-abusive-eslint-disable': 'error',
+    'unicorn/no-array-instanceof': 'error',
+    'unicorn/no-console-spaces': 'error',
+    'unicorn/no-fn-reference-in-iterator': 'off',
+    'unicorn/no-for-loop': 'error',
+    'unicorn/no-hex-escape': 'error',
+    'unicorn/no-new-buffer': 'error',
+    'unicorn/no-process-exit': 'error',
+    'unicorn/no-unreadable-array-destructuring': 'error',
+    'unicorn/no-unsafe-regex': 'off',
+    'unicorn/no-unused-properties': 'error',
+    'unicorn/no-zero-fractions': 'error',
+    'unicorn/number-literal-case': 'error',
+    'unicorn/prefer-exponentiation-operator': 'error',
+    'unicorn/prefer-includes': 'error',
+    'unicorn/prefer-node-remove': 'error',
+    'unicorn/prefer-query-selector': 'error',
+    'unicorn/prefer-starts-ends-with': 'error',
+    'unicorn/prefer-text-content': 'error',
+    'unicorn/prefer-type-error': 'error',
+    'unicorn/prevent-abbreviations': [
+      'error',
+      {
+        replacements: {
+          ref: false,
+          refs: false,
+          prop: false,
+          props: false,
+          src: false,
+          param: false,
+          params: false,
+          args: false,
+        },
+      },
+    ],
+    'unicorn/throw-new-error': 'error',
+  },
+  overrides: [
+    {
+      files: ['*.vue'],
+      parser: 'vue-eslint-parser',
+      rules: {
+        'prettier/prettier': ['off'],
+      },
+    },
+    {
+      files: ['*.ts'],
```

Comment:
Don't forget those nice definition files :)
```suggestion
      files: ['*.ts', '*.d.ts'],
```

Reply:
Should we turn off `@typescript-eslint/no-unused-vars` for definition files?

Reply:
Why would we need to explicit specify `.d.ts` files? They have a `.ts` extension as well right?

Reply:
hmm you're right, not sure if I actually got errors, or that I just thought it was needed :D